tests/php/View/AttributesHTMLTest.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace SilverStripe\View\Tests;
+
+use SilverStripe\Dev\SapphireTest;
+
+class AttributesHTMLTest extends SapphireTest
+{
+
+    public function singleAttributeProvider(): array
+    {
+        return [
+            'empty string' => ['test', '', 'Empty string is not converted to a different falsy value'],
+            'Zero' => ['test', 0, 'Zero is not converted to a different falsy value'],
+            'Null' => ['test', 0, 'Null is not converted to a different falsy value'],
+            'False' => ['test', false, 'False is not converted to a different falsy value'],
+            'Empty array' => ['test', [], 'Empty array is not converted to a different falsy value'],
+            'True' => ['test', true, 'True is stored properly as an attribute'],
+            'String' => ['test', 'test', 'String is stored properly as an attribute'],
+            'Int' => ['test', -1, 'Int is stored properly as an attribute'],
+            'Array' => ['test', ['foo' => 'bar'], 'Array is stored properly as an attribute'],
+        ];
+    }
+
+    /** @dataProvider singleAttributeProvider */
+    public function testGetAttribute($name, $value, $message): void
+    {
+        $dummy = new AttributesHTMLTest\DummyAttributesHTML();
```

Comment:
```suggestion
        $dummy = DummyAttributesHTML();
```

Import this class.  Make this change throughout this file